src/Api/Environments.php

```diff
@@ -54,7 +54,7 @@ public function create($project_id, array $parameters = [])
         $resolver->setDefined('external_url')
```

Comment:
There is a typo in your commit message, `Environemnts` should be `Environments`

Reply:
there is also a typo in his reply to Graham. It's `amateur` not `amatuer`